BookClub/views/clubs.py

```diff
@@ -0,0 +1,27 @@
+'''Club Related Views'''
+from django.shortcuts import render
+from django.urls import reverse_lazy
+from django.contrib.auth.mixins import LoginRequiredMixin
+from django.views.generic.edit import CreateView, DeleteView, UpdateView
+
+from BookClub.models.club import Club
+from BookClub.forms.club import ClubForm
+
+from BookClub.models.club_membership import ClubMembership
+
+def club_dashboard(request):
+    """This is the club dashboard view."""
+    return render(request, 'club_dashboard.html')
+
+class CreateClubView(LoginRequiredMixin, CreateView):
```

Comment:
We should add a message to form_valid for success and add a form_invalid so that we can add an error message

Reply:
Done, pls check